packages/gatsby-plugin-typescript/README.md

```diff
@@ -1,6 +1,6 @@
 # gatsby-plugin-typescript
 
-Provides drop-in support for TypeScript and TSX.
+Allows Gatsby to build TypeScript and TSX files. Does NOT run type checking during build (see Caveats).
```

Comment:
I would even go so far as to bold the second sentence. This fact was a big pain point for my team.

Reply:
Why not run type-check as part of CI?

Reply:
you should! but this plugin doesn't do it